### PR TITLE
Docs: Code Sandbox example on homepage

### PR DESCRIPTION
**Pull request by carwack**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been reviewed and added / updated if needed (for bug fixes
      /start features)

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [X] Documentation content changes
- [ ] Other (please describe):

## What is the new behavior?

- Add Iframe to Dom Elements list
- Add Code Sandbox example to home page

Screenshot 2021-04-01 at 14 11 15

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## Other information
TODO: Update the Example with a Chakra UI Vue Chidori V1 example instead of Chakra Ui Vue v0

### CODE REVIEW

File: packages/system/src/system.utils.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export const domElements = [
   'h6',
```
Comment (codebender828) [inline packages/system/src/system.utils.ts:35]: Nice work here. Please remember to add a changeset with a minor bump for all changes made to the core components.

**Review by codebender828 — CHANGES_REQUESTED** [review]
Thanks, @carwack !

Please include a changeset with a patch bump to the changed packages. Since this PR makes changes to the core library, it's important to version those changes.

You can include one by running `yarn changeset`

File: packages/system/src/system.utils.ts (hunk 1 above)
Comment (carwack) [reply packages/system/src/system.utils.ts:35]: Hi, Thanks for the feedback!

So if I'm right, I should run `yarn changeset` and then include all changed packages in a minor bump? 😄

**Review by codebender828 — APPROVED** [review]
👍🏽

**Pull request merged**